<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vegetable</class>
 <widget class="QWidget" name="vegetable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>1380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>1380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1141</width>
     <height>1431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/vegetable_page/vegetable_page_back.png);</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>452</width>
      <height>175</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>452</width>
      <height>175</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/vegetable_page/click_off/t_1.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>770</y>
      <width>452</width>
      <height>175</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>452</width>
      <height>175</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/vegetable_page/click_off/t_2.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>590</y>
      <width>452</width>
      <height>175</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>452</width>
      <height>175</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/vegetable_page/click_off/t_5.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>950</y>
      <width>452</width>
      <height>175</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>452</width>
      <height>175</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/vegetable_page/click_off/t_4.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>452</width>
      <height>175</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>452</width>
      <height>175</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/vegetable_page/click_off/t_3.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>290</y>
      <width>452</width>
      <height>740</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>452</width>
      <height>740</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/vegetable_page/more_variety.png);</string>
    </property>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>385</width>
       <height>175</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>385</width>
       <height>175</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/image/vegetable_page/click_off/t_6.png);</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>360</y>
       <width>385</width>
       <height>175</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>385</width>
       <height>175</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/image/vegetable_page/click_off/t_8.png);</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>540</y>
       <width>385</width>
       <height>175</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>385</width>
       <height>175</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/image/vegetable_page/click_off/t_7.png);</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
